### PR TITLE
Document how to opt-out of LZMA Artifact compression

### PR DESCRIPTION
**Pull request by lluiscampos**
Changelog: None

Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

**Comment by lluiscampos** [comment]
In theory, we should also have a similar change in `mender-artifact`, but there the README file has no instructions for building the CLI tool. So I think it is acceptable to ignore there the LZMA detail

**Comment by estenberg** [comment]
Change makes sense.

However, slightly related we have a cross-compiling doc here: https://docs.mender.io/client-configuration/cross-compiling. Should we remove cross compiling from this doc and just point to the docs page?

**Comment by lluiscampos** [comment]
@estenberg @kacf @mirzak @apodogrocki

Right, so again we have duplicated instructions (but slightly different!) to do the same thing :weary: 

IMO, instructions for compiling (and cross-compiling) should be as close to the code as possible (i.e. README.md in the same repo as the code). If you agree with it, I will then rework for the mender-docs to point to here.

However, I would like to wait for the outcome of [MEN-2399](https://tracker.mender.io/browse/MEN-2399) to use then the same toolchain.

That I can think of, we have cross-compile recipes in several places with diferent tool-chains and including or not lzma dependency
* mender-convert: `gcc-arm-linux-gnueabihf` from debian (WIP migration to `armv6-eabihf-glibc-stable-2018.11-1` from bootlin)
* mender-dist-packages: `gcc-arm-linux-gnueabihf` from debian
* mender README.md: `gcc-linaro-arm-linux-gnueabihf-raspbian-x64` from raspberrypy/tools (obs. this should be changed no matter what!)
* mender-docs: `gcc-arm-linux-gnueabihf` from debian

On top of that, some of them have instructions for the liblzma dependency (it will be included once we merge update_modules fork) while others not.

So, I suggest we don't block this PR and we unify the procedures after MEN-2399

File: README.md [hunk 1]
```diff
@@ -73,6 +73,15 @@ issue. We thank you in advance for your cooperation.
 * [Go compiler](https://golang.org/dl/)
 * liblzma-dev package
 
+#### LZMA support opt-out
+
+If no LZMA Artifact compression support if desired, you can ignore the `liblzma-dev` package
+dependency and substitue the `make` commands in the instructions below for:
```
Comment (kacf) [inline README.md]: `s/substitue/substitute/`

Comment (lluiscampos) [reply README.md]: Typo fixed :+1:

**Pull request merged**